docs/en/reference/working-with-associations.rst

```diff
@@ -500,6 +500,19 @@ and the "remove" operation.
         //...
     }
 
+Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
+association, you can now create a user and persist his comment as follows:
```

Comment:
> You can't call a person "it"! I followed the convention on this page, referring to a user as "he".

Although I agree with you, there's an entire social movement (and a quite aggressive one) about making stuff gender neutral, and I just want to avoid them bother the project due to a misplaced word here and there.

 > The example is about just one comment, not many.

Yes, but a user always has multiple comments, regardless if it's 0, 1 or 10000.